features/help_me_choose.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Help me choose
+
+  As a user, i want to use search filters, because they give a quick and easy way to find a car that suits me.
+
+  Background:
+    Given help me choose page is opened
+
+  @bodystyle
```

Comment:
Why do you need this and next tags?

Reply:
For possibility to run every scenario separately.

Reply:
There is no need in this option. Tags are used to mark scenarios that should be run together. If you need to run single scenario, you can do this in RubyMine - right-click on scenario and choose run